### DISCUSSION TITLE
How to convert Euler orientation to corresponding miller indices?

### DISCUSSION THREAD

**Discussion by Mustafa-Rifat** [mtex-toolbox/mtex#956] [Ask Anything] [answered]
Hello,

If I have an orientation defined in Euler convention, is there a way to define that in miller indices i.e. (hkl)[uvw] format like the attached?
![image](https://user-images.githubusercontent.com/60717121/114585946-252eaf80-9c52-11eb-9398-7f9985c01da9.png)
Thank you!

**Comment by kilir** [comment, marked as answer]
Hi,
`orientation/round2Miller` should do that.
Cheers,
Rüdiger

**Reply by GH4169** [reply]
Thanks
